backstop/tests/table.html

```diff
@@ -110,94 +113,91 @@ <h2 id="default">Default</h2>
                     viewBox="0 0 16 16"
```

Comment:
generic-iui-icons ???
In this file, also in file-upload...

Reply:
No idea how that one happened, fixed now.